compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -372,44 +359,63 @@ class ROCMTargetBackend final : public TargetBackend {
       // multi-threading issues.
       llvm::LLVMContext context;
 
-      auto llvmModule =
-          mlir::translateModuleToLLVMIR(innerModuleOp, context, libraryName);
-      if (!llvmModule) {
-        return variantOp.emitError() << "failed to translate the MLIR LLVM "
-                                        "dialect to the native llvm::Module";
-      }
-
+      // Set up attributes so upstream's conversions work right.
+      auto *rocdlDialect =
+          innerModuleOp.getContext()->getLoadedDialect<ROCDL::ROCDLDialect>();
+      auto kernelAttrHelper = rocdlDialect->getKernelAttrHelper();
+      auto flatWgAttrHelper = rocdlDialect->getFlatWorkGroupSizeAttrHelper();
+      auto reqdWgSizeAttrHelper =
```

Comment:
What is `reqd`? Required?

Reply:
Yep, and that's an LLVM name so there's not much I can do about it